### DISCUSSION TITLE
"outline_polygon" value problem in PCB creation

### DISCUSSION THREAD

**Discussion by giannoko** [ansys/pyaedt#5377] [Q&A]
Hi,

I want to create a PCB component using the "create_pcb_from_3dlayout" function. I want to use the "Polygon" as the "extend_type" but if I set "poly_0" as the value of the "outline_polygon" (as pyaedt example suggests: https://examples.aedt.docs.pyansys.com/version/dev/examples/electrothermal/ecad_import.html) it results in an error saying that "poly_0" is not a correct value. I realized that "outline_polygon" value depends on ecad file. For example, I got "poly__3098" and "poly__4011" values in "outline_polygon" for two different ecad files. 
How can I set a general "outline_polygon" value, valid for every ecad file?

Thank you

**Comment by hui-zhou-a** [comment]
Hi @anskgiannok, you can get the name of the polygon on outline layer from below

![image](https://github.com/user-attachments/assets/a23410f0-6ca7-4ee5-9f59-b6ed10e63d82)

**Reply by giannoko** [reply]
Hi @hui-zhou-a. Thank you for your answer but what is "edbapp" ?

**Comment by hui-zhou-a** [comment]
It is  your board.

from pyedb import Edb

edbapp = Edb("your_edb.aedb")

**Reply by giannoko** [reply]
It worked. Thank you!
